regular_expressions_groups/regular_expressions_groups.md

```diff
@@ -0,0 +1,280 @@
+
+So, what can we do? How can we indicate where the "or" stops?  We can use parentheses for grouping as a way to create a boundary.
+
+`(WT|PBMC)\-\d{5,6}` would work, and would be interpreted as:
+
+* "WT" or "PBMC" (and please capture whichever it is)
+* Then a hyphen
+* Then 5-6 digits
+
+[That expression will work as valid regex](https://regex101.com/r/ZHs5iI/1) that will match with your various mouse subject ids.  However, you really don't **need** to capture that group, you're just using the parentheses to enclose a group that could go one of two different ways.  Let's consider another option on the next page: a **non-capturing group**.  
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+You might wonder what would happen if we added a third mouse type, like a CD34+ humanized mouse model with the code "CD34" in its identifier.  Well, we can string together as many or statements as we need.  We can indicate WT or PBMC or CD34 by using `WT|PBMC|CD34`.  This indicates that any of the three values separated by the pipes are valid at that location in a pattern.
+
+</div>
+
+## Non-Capturing Groups
+
+So far, we have used the vertical pipe (`|`) and a capturing group to describe the pattern of our mouse identifiers:  `(WT|PBMC)\-\d{5,6}`.
+
+But what if you don't need to capture the mouse type?  You might just want to make sure that the identifier is valid and matches the pattern correctly, without extracting any text from the identifier.
+
+If that's the case, you can indicate that it's a **non-capturing group** by adding `?:` after the opening parenthesis, like this:
+
+`(?:WT|PBMC)\-\d{5,6}`
+
+[This will also work](https://regex101.com/r/ko7ySn/1) as a matching regular expression, but won't capture the group in question.
+
+This may seem unnecessary.  Why go to the trouble of creating a non-capturing group?  You could simply use a capturing group but not pay any attention to what you capture.  That's true, but consider that in a long string, you might actually want to capture some data and not capture other data, and getting an extra captured group could be confusing or annoying.  It's useful to be able to group things and **not** capture them.
+
+
+### Quiz: Non-Capturing Groups and the Logical "Or"
+
+You're looking through lab notes written by a colleague who uses codes and abbreviations liberally.  
+
+This investigator is known to write their lowercase initials ("sjp") and a single space followed by either "help" (always in lowercase) or "911" in notes to highlight important potential pitfalls.  
+
+Using the logical "or" and using a non-capturing group (remember: the syntax is `(?:)`), write a regular expression that would allow you to capture any of the "sjp help" or "sjp 911" references.
```

Comment:
```suggestion
Using the logical "or" and using a non-capturing group (remember: the syntax is `(?:)`), write a regular expression that would match both the "sjp help" and "sjp 911" patterns.
```

I think you were using a colloquial "capture" here mixed in with the technical "non-capturing"